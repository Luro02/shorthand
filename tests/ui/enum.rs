use shorthand::ShortHand;

#[derive(ShortHand)]
enum ExampleEnum {
    First,
    Second,
    Third { a: String, b: usize },
}

fn main() {}
