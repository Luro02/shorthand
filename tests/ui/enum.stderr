error: enum are not supported.
 --> $DIR/enum.rs:4:1
  |
4 | / enum ExampleEnum {
5 | |     First,
6 | |     Second,
7 | |     Third { a: String, b: usize },
8 | | }
  | |_^
